time my_time
---
bool is_newer
time current_time